src/tests/Microsoft.PowerFx.Core.Tests/FunctionInfoTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class FunctionInfoTests
+    {
+        [Fact]
+        public void FunctionInfo()
+        {
+            // Get any function info. 
+            var engine = new Engine();
+            var infos = engine.FunctionInfos.ToArray();
+            var infoAbs = infos.Where(info => info.Name == "Mid").First();
```

Comment:
>infoAbs [](http://example.com/codeflow?start=16&length=7)

Why not call it `infoMid` since you are testing the Mid function, not Abs?

Reply:
fixed - I originally started with Abs, but Mid is more interesting.